_includes/code/animation/shared_element_transition/main.dart

```diff
@@ -43,26 +43,26 @@ class SharedElementTransition extends StatelessWidget {
 
     return new Scaffold(
       appBar: new AppBar(
-        title: const Text('Main Page'),
+        title: const Text('Shared Element Transition'),
       ),
       body: new Center(
         child: new PhotoHero(
-          photo: 'images/curly.jpg',
+          photo: 'images/flippers-alpha.png',
           width: 300.0,
           onTap: () {
             Navigator.of(context).push(
                 new MaterialPageRoute<Null>(builder: (BuildContext context) {
               return new Scaffold(
                 appBar: new AppBar(
-                  title: const Text('Detail Page'),
+                  title: const Text('Flippers Page'),
                 ),
                 body: new Container(
                   // Use background color to emphasize that it's a new page.
                   color: Colors.lightBlueAccent,
                   padding: const EdgeInsets.all(16.0),
                   alignment: FractionalOffset.topLeft,
                   child: new PhotoHero(
-                    photo: 'images/curly.jpg',
+                    photo: 'images/flippers-alpha.png',
                     width: 100.0,
                     onTap: () {
                       Navigator.of(context).pop();
```

Comment:
You could put a comment here that reminds the reader that the pop operation triggers a Hero flight. Likewise for the earlier push